docs/modules/ROOT/pages/tutorials/install-cloudscale.adoc

```diff
@@ -0,0 +1,68 @@
+= Install Cloudscale Stack
+
+This tutorial will guide you through the installation process of installing the following parts:
+
+- A Crossplane provider to manage cloudscale.ch resources.
+//- A Crossplane composite (XRD) which serves as the user's "interface" (API) for ordering S3 buckets.
+//- A Crossplane composition that "implements" an XRD using the cloudscale.ch provider.
```

Comment:
Note: I haven't tested this tutorial yet. I will fix missing or wrong steps when we roll it out, in a different story.